assets/javascripts/discourse/controllers/admin-plugins-discourse-subscriptions-subscriptions.js.es6

```diff
@@ -1,12 +1,30 @@
+import { alias } from "@ember/object/computed";
+import AdminSubscription from "discourse/plugins/discourse-subscriptions/discourse/models/admin-subscription";
 import Controller from "@ember/controller";
 import showModal from "discourse/lib/show-modal";
 
 export default Controller.extend({
+  loading: false,
+  canLoadMore: alias("model.has_more"),
```

Comment:
You probably don't need a CP for this. Your code below can say `this.model.has_more` and save some ceremony.

Reply:
Good catch. This was a holdover from a previous iteration.